queue/src/fiskaltrust.Interface.Tagging/Models/V1/DE/ftPayItemCases.cs

```diff
@@ -0,0 +1,34 @@
+﻿using fiskaltrust.ifPOS.v1;
+using fiskaltrust.Interface.Tagging.Generator;
+
+namespace fiskaltrust.Interface.Tagging.Models.V1.DE
+{
+    [CaseExtensions(OnType = typeof(PayItem), OnField = nameof(PayItem.ftPayItemCase), Mask = 0xFFFF, Prefix = "V1")]
+    public enum ftPayItemCases : long
+    {
+        UnknownPaymentType0x0000 = 0x0000,
+        Cash0x0001 = 0x0001,
+        CashForeignCurrency0x0002 = 0x0002,
+        CrossedCheque0x0003 = 0x0003,
+        DebitCard0x0004 = 0x0004,
+        CreditCard0x0005 = 0x0005,
+        Online0x0006 = 0x0006,
```

Comment:
Online should be 0x0007, this one is missed :  MultipurposeVoucher0x0006 = 0x0006,